docs/specs/moniker.yml

```diff
@@ -831,3 +831,110 @@ outputs:
     }
   build.log: |
     ["error","publish-url-conflict","Two or more documents of the same version('netcore-1.0') publish to the same url '/docs/toc.json': 'docs/v1/TOC.md', 'docs/v2/TOC.md'"]
+---
+# Build resource file with moniker
+inputs:
+  docfx.yml: |
+    monikerRange:
+      'docs/v1/**/*.md': '>= netcore-1.0'
+    routes:
+      docs/v1/: docs/
+    monikerDefinition: monikerDefinition.json
+  docs/v1/a.md: |
+    ---
+    monikerRange: netcore-1.0 || netcore-1.1
+    ---
+    Moniker: netcore-1.0, netcore-1.1
+    ![image](c.png)
+  docs/v1/b.md: |
+    ---
+    monikerRange: netcore-1.1 || netcore-1.2
+    ---
+    Moniker: netcore-1.1, netcore-1.2
+    ![image](c.png)
+  docs/a.md: |
+    No version
+    ![image](c.png)
+  docs/v1/c.png:
+  docs/c.png:
```

Comment:
should we support this? what's the behavior if there are 2 articles with the same url, one with version, the other does not?
```
docs/v1/a.md
docs/a.md
```